Content.Shared/Strip/Components/SharedStrippableComponent.cs

```diff
@@ -77,4 +78,24 @@ public StrippingBoundUserInterfaceState(Dictionary<(string ID, string Name), str
             Handcuffs = handcuffs;
         }
     }
+
+    /// <summary>
+    /// Used to modify strip times.
+    /// </summary>
+    [NetSerializable, Serializable]
+    public sealed class BeforeStripEvent : EntityEventArgs, IInventoryRelayEvent
+    {
+        public readonly float InitialTime;
+        public float Time;
+        public float Coeff = 1;
+        public float Additive = 0;
```

Comment:
I think supporting both is kinda jank and it should just be one or the other for adding vs multiplying.